src/pages/home/report/ReportActionItem.tsx

```diff
@@ -623,6 +623,9 @@ function ReportActionItem({
             children = <ReportActionItemBasicMessage message={translate('iou.unheldExpense')} />;
         } else if (action.actionName === CONST.REPORT.ACTIONS.TYPE.MERGED_WITH_CASH_TRANSACTION) {
             children = <ReportActionItemBasicMessage message={translate('systemMessage.mergedWithCashTransaction')} />;
+        } else if (action.actionName === CONST.REPORT.ACTIONS.TYPE.ROOM_CHANGE_LOG.UPDATE_ROOM_DESCRIPTION) {
```

Comment:
@ishpaul777 check out all the `is$ACTION_TYPEAction` utility functions in ReportActionUtils. You can use type predicates to narrow the type of a report action, and thus avoid having to do any type casting.